pkg/registry/apis/iam/resourcepermission/queries/permission_insert.sql

```diff
@@ -0,0 +1,11 @@
+INSERT INTO {{ .Ident .PermissionTable }} (role_id, action, scope, created, updated, kind, attribute, identifier)
+VALUES (
+    {{ $.Arg $.RoleID }},
+    {{ $.Arg $.Permission.Action }},
+    {{ $.Arg $.Permission.Scope }},
+    {{ $.Arg $.Now }},
+    {{ $.Arg $.Now }},
+    {{ $.Arg $.Permission.Kind }},
+    {{ $.Arg $.Permission.Attribute }},
+    {{ $.Arg $.Permission.Identifier }}
```

Comment:
nit: are the `$` necessary?

Reply:
You are correct, they are not. It's a left over from a copy paste.